src/components/organisms/Modal/Modal.module.scss

```
@@ -47,26 +41,28 @@
   @include flexCenter;
   width: 122px;
   padding: 11px 0px;
+
+  border-radius: 10px;
+  background: $darkGray500;
+
   color: #ff453a;
   font-size: 1.6rem;
   font-weight: 610;
```

```suggestion
  font-weight: $medium;
```
여기 그냥 600 으로 통일하는 거 어때용

---

좋습니다👍🏻